### PR TITLE
feat: symbol support

### PR DESCRIPTION
**Pull request by jgillick**
Starting with ECMAScript 2015, symbol is a primitive data type, just like number and string. For the sake of JSON schema and validation, it can be cast to a string. 

However, in reality, the symbol can be constructed with anything. Is there a better way to reflect this? In this case, would it make sense for `symbol` to be converted into an `anyOf` validation?

```
"anyOf": [
    { "type": "null" },
    { "type": "boolean" },
    { "type": "object" },
    { "type": "array" },
    { "type": "number" },
    { "type": "string" }
  ]
```

In my case, the type `symbol` is used to define a property key on an object: `Record<symbol, string>;`.

### CODE REVIEW

**Comment by domoritz** [comment]
The way to express any in JSON schema is with an empty object `{}`. Do you think that's better than string? Either way, can you add a test case?

File: package.json [hunk 1]
```diff
@@ -79,6 +79,7 @@
     "test": "jest test/ --verbose",
     "test:fast": "FAST_TEST=1 jest test/ --verbose",
     "debug": "node -r ts-node/register --inspect-brk ts-json-schema-generator.ts",
-    "run": "ts-node ts-json-schema-generator.ts"
+    "run": "ts-node ts-json-schema-generator.ts",
+    "prepare": "yarn build"
```
Comment (domoritz) [inline package.json:83]: Why is that needed? `ts-node` works without building the project.

Comment (jgillick) [reply package.json:83]: I'll remove it. I added that to get it building when installing directly from github in the package.json dependencies.

**Comment by jgillick** [comment]
Per the empty object: that makes sense. Let's try that route.

**Comment by domoritz** [comment]
Sweet. Thanks for your contribution to the project.

**Comment by jgillick** [comment]
I've updated the code to cast `symbol` to `any` and added a simple unit test. Let me know if this is sufficient. Thank you for the tool.
